git-config/tests/integration_tests/file_integeration_test.rs

```diff
@@ -76,7 +76,17 @@ fn get_value_for_all_provided_values() -> crate::Result {
     assert_eq!(
         file.value::<git_config::values::Path>("core", None, "location")?,
         git_config::values::Path {
-            value: Cow::Borrowed(b"/tmp")
+            value: Cow::Borrowed(
+                format!(
+                    "{}/tmp",
+                    dirs::home_dir()
+                        .ok_or("empty home dir")?
```

Comment:
It's better to `expect("msg")` here instead of substituting values which are then seemingly the expected value.